src/app.ts

```diff
@@ -38,4 +53,18 @@ app.use(function (err: any, res: Response) {
 	res.end(err.message);
 });
 
+app.use(function (req: any, res: Response, next: NextFunction) {
```

Comment:
What is this middleware for?

Reply:
Ya it's for some testing purposes but without that middleware also swagger will work I have just tested it out and has removed , Basically that was used to tell the page to allow these types of headers to be accepted in post request but now they are not needed at all.